app/src/main/java/com/example/gradeassist/ui/notifications/NotificationsFragment.kt

```diff
@@ -28,4 +30,11 @@ class NotificationsFragment : Fragment() {
         })
         return root
     }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (requestCode == Activity.RESULT_OK){
+            //TODO Add the needed onActivityResult if statements with resultCodes  needed for this fragments
```

Comment:
Ditto comment above.  

This block of code is duplicated three times in this review, which violates the "duplicated blocks" technical debt rule.  A better approach: if this function is required in all three fragments, it's best to create a superclass fragment, from which each of these fragments inherit, and place the block there.  Then the function can be inherited by child classes.

As it is, though, the function is not being used, and can safely be removed.